vnext/include/ReactUWP/Views/ViewManagerBase.h

```diff
@@ -62,6 +62,7 @@ class REACTWINDOWS_EXPORT ViewManagerBase : public facebook::react::IViewManager
 
   virtual void UpdateProperties(ShadowNodeBase *nodeToUpdate, const folly::dynamic &reactDiffMap);
 
+  virtual void DispatchCommand(XamlView viewToUpdate, const std::string &commandId, const folly::dynamic &commandArgs);
   virtual void DispatchCommand(XamlView viewToUpdate, int64_t commandId, const folly::dynamic &commandArgs);
```

Comment:
Why do we need to suport the older way? There are exactly 2 existing commands in the entire core, both on scrollviewer. Update scrollviewer to the new way and completely delete the old way.

Reply:
I originally kept it because both still exist upstream (one marked for eventual removal), but I think you're right that we could just remove the old since we have control of our view managers and the string constant command trick.